tests/test_recoder.py

```diff
@@ -28,13 +28,13 @@ def boost():
     assert hyperspace.module.boost() == 200
     rec.commit()
 
-    after = open(hyperspace.main.filename).read()
-    expected = open(hyperspace.cf.recoded.filename).read()
+    after = open(hyperspace.main.filename, encoding="utf8").read()
+    expected = open(hyperspace.cf.recoded.filename, encoding="utf8").read()
```

Comment:
These don't contain Unicode characers AFAIK.

Reply:
> These don't contain Unicode characers AFAIK.

Sorry, I will not fix this bug.